packages/test/src/components/Sidebar/LogoutButtons.tsx

```
@@ -1,39 +1,48 @@
-import { Button } from '../common/Button'
+import { Button } from '../common/Button';
 import { useAccount } from 'wagmi';
 import { useDisconnect } from 'wagmi';
 import { useBitcoinWallet } from '@gardenfi/wallet-connectors';
+import { useWalletStore } from '../../store/useWalletStore';
```

ESLint

[error] 'useWalletStore' is defined but never used.

(@typescript-eslint/no-unused-vars)